This isn't Linux, we actually execute EXEs directly. Unfortunately there
seems to be no way to exclude ELF binaries (no specific subtype).

--- origsrc/shared-mime-info-1.2/freedesktop.org.xml.in	2013-09-30 05:01:23.000000000 -0500
+++ src/shared-mime-info-1.2/freedesktop.org.xml.in	2014-03-21 02:26:50.418199000 -0500
@@ -2562,6 +2562,7 @@ command to generate the output files.
   </mime-type>
   <mime-type type="application/x-ms-dos-executable">
     <_comment>DOS/Windows executable</_comment>
+    <sub-class-of type="application/x-executable"/>
     <generic-icon name="application-x-executable"/>
     <magic priority="50">
       <match type="string" value="MZ" offset="0"/>
